text/0006-token-api.md

```diff
@@ -0,0 +1,92 @@
+- Feature Name: token_api
+- Start Date: 2021-08-04
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
+
+# Summary
+[summary]: #summary
+
+Token API will provide token information by requesting `hathor-wallet-service`.
+
+# Motivation
+[motivation]: #motivation
+
+We need to retrieve token data without requesting the full node. Wallet service holds those data, so, we'll use it for now. 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+- We'll basically act as a proxy making requests to wallet service
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+**Endpoints:**
+
+GET `/token/:id` return `200` | `404`
+
+A request is made for Wallet service passing the same id
```

Comment:
This explanation should be in the low level design. What is the API that will be accessed? Maybe add a link its documentation here.